packages/bpk-component-accordion/src/BpkAccordionItem.js

```diff
@@ -134,16 +134,19 @@ const BpkAccordionItem = (props: Props) => {
           className={getClassName('bpk-accordion__toggle-button')}
         >
           <span className={getClassName('bpk-accordion__flex-container')}>
-            <BpkText
-              textStyle={textStyle}
-              tagName={tagName}
-              className={titleTextClassNames.join(' ')}
-            >
-              {clonedIcon}
-              {title}
-            </BpkText>
+          <span className={titleTextClassNames.join(' ')}>
+              <BpkText
+                textStyle={textStyle}
+                tagName={tagName}
+              >
+                {clonedIcon}
+                {title}
+              </BpkText>
+            </span>
             <span className={getClassName('bpk-accordion__icon-wrapper')}>
-              <ExpandIcon className={iconClassNames.join(' ')} />
+              <div className={iconClassNames.join(' ')}>
```

Comment:
be careful when putting `div` tags inside of spans. The HTML spec likely doesn't allow it, https://stackoverflow.com/questions/11314668/what-elements-can-a-span-tag-contain-in-html5

Reply:
Replaced with a div, no negative effects except the already happening vertical space problem